app/models/tag.rb

```diff
@@ -49,6 +50,13 @@ class Tag < ActiveRecord::Base
   scope :only_children, -> { where('parent_id IS NOT NULL') }
   scope :in_alphabetical_order, -> { order('title ASC') }
 
+  # The links last sent to the content-store.
+  serialize :published_groups, JSON
+
+  after_initialize do
+    self.published_groups ||= []
+  end
```

Comment:
Instead of using `after_initialize` here (which can get quite expensive), we could set the default value in the schema to be the serialized representation of empty array.